src/workloads/issues/MongosLatency.yml

```diff
@@ -0,0 +1,233 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload was created to reproduce the issue described in HELP-25757 / SERVER-58997. The goal
```

Comment:
Should we switch to only reference the SERVER ticket here, since it is the public project?

Reply:
good idea, I'll update.